include/ruby/internal/intern/process.h

```diff
@@ -30,6 +30,16 @@ RBIMPL_SYMBOL_EXPORT_BEGIN()
 
 /* process.c */
 
+/**
+ * Creates a new instance of Process::Status.
+ *
+ * @param[in] pid The process ID.
+ * @param[in] status The process status.
+ * @param[in] error Error code.
```

Comment:
Could this be documented better? It's the error code corresponding to which system call, etc.

Reply:
Fixed in d30aa0c654dd34e666fb9c493b19bbddda4cce40.